clsp/utils.clib

```diff
@@ -5,22 +5,9 @@
   ; sha256
   (include sha256tree.clib)
 
-  (defconstant b32 32)
-
-  (defun-inline size_b32 (var)
-    (= (strlen var) b32)
-  )
-
-  (defun calculate_coin_id (parent puzzlehash amount)
-    (if (all (size_b32 parent) (size_b32 puzzlehash) (> amount -1))
-      (sha256 parent puzzlehash amount)
-      (x)
-    )
-  )
-
   ;; return the full puzzlehash for a singleton with the innerpuzzle curried in
   (defun-inline calculate_full_puzzle_hash (SINGLETON_STRUCT inner_puzzle_hash)
-    (curry_hashes_inline (f SINGLETON_STRUCT)
+    (curry_hashes (f SINGLETON_STRUCT)
```

Comment:
Docs say it is sometimes more efficient to use inline. After profiling, realized this wasn't one of those times.